src/containers/admin-dashBoard/UpdateTeamModal.tsx

```diff
@@ -5,10 +5,23 @@ import { TFunction, useTranslation } from 'react-i18next';
 import { toast } from 'react-toastify';
 import Button from '../../components/Buttons';
 import { Team, Cohort } from './Teams';
+import ControlledSelect from '../../components/ControlledSelect';
 
 export const UpdateTeam = gql`
-  mutation UpdateTeam($updateTeamId: ID!, $name: String, $orgToken: String) {
-    updateTeam(id: $updateTeamId, name: $name, orgToken: $orgToken) {
+  mutation UpdateTeam(
+    $updateTeamId: ID!
+    $name: String
+    $cohort: String
+    $TTL: String
```

Comment:
@JacquelineTuyisenge read the descrption of PR ,  I have said on that issues that admin can not change phase of team unless he/she change it on cohort , this is because team are getting coordinator ,phase,program from cohort 

change the cohort and see how the phase are being change too 

thanks